src/content/docs/en/guides/images.mdx

```diff
@@ -8,10 +8,12 @@ import Since from '~/components/Since.astro';
 
 Astro provides several ways for you to use images on your site, whether they are stored locally inside your project, linked to remotely, or stored in a CMS or CDN!
 
-:::note[using Assets (Experimental)]
-Experimental asset support was added in `astro@2.1.0`. If you have enabled assets support, see the [Assets (Experimental) Guide](/en/guides/assets/) for extra information.
+:::note[`astro:assets` (Experimental - coming in v3.0)]
+The experimental `astro:assets` module will be built in to `astro@3.0`.
```

Comment:
Experimental assets is technically already "built-in", it's just under a flag. Maybe "enabled by default" or something similar would be better?